dll/ntdll/def/ntdll.spec

```diff
@@ -881,7 +881,7 @@
 @ stdcall RtlGetFrame()
 @ stdcall RtlGetFullPathName_U(wstr long ptr ptr)
 @ stdcall RtlGetFullPathName_UstrEx(ptr ptr ptr ptr ptr ptr ptr ptr)
-@ stub -version=0x600+ -arch=x86_64 RtlGetFunctionTableListHead
+@ stdcall -version=0x600+ -arch=x86_64 RtlGetFunctionTableListHead()
```

Comment:
This one seems to be present in the ntdll of WinXP x64.
See also https://web.archive.org/web/20210825055930/https://www.geoffchappell.com/studies/windows/win32/ntdll/history/names52.htm
> RtlGetFunctionTableListHead | begins from SP1; x64 only; undocumented until 2004-2006; documentation withdrawn in 2007-2008

(the part about "doc withdrawn" is questionable, since it's in MSDN).